File: 88. Merge Sorted Array/88.MergeSortedArray.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        for (int i = m; i < m + n; ++i) {
            nums1[i] = nums2[i - m];
        }

        // Bubble sort in place nums1
        for (int i = 0; i < nums1.size(); ++i) {
            for (int j = i + 1; j < nums1.size(); ++j) {
                if (nums1[i] > nums1[j]) {
                    std::swap(nums1[i], nums1[j]);
                }
            }
        }
    }
};

File: 88. Merge Sorted Array/88.MergeSortedArray.js
var merge = function (nums1, m, nums2, n) {
  for (let i = m; i < m + n; i++) {
    let temp = nums2.shift();
    nums1[i] = temp;
  }

  //buble sort in place nums1
  for (let i = 0; i < nums1.length; i++) {
    for (let j = i + 1; j < nums1.length; j++) {
      if (nums1[i] > nums1[j]) {
        let temp = nums1[i];
        nums1[i] = nums1[j];
        nums1[j] = temp;
      }
    }
  }
};
